data_validation/query_builder/query_builder.py

```diff
@@ -508,7 +512,7 @@ def compile(self, data_client, schema_name, table_name):
                 self.compile_aggregate_fields(filtered_table)
             )
         else:
-            query = grouped_table
+            query = filtered_table
```

Comment:
Wouldn't we need to return grouped_table in case we want to add aggregate fields or a group_by field?

Reply:
good catch. Reverting.